ratis-server/src/main/java/org/apache/ratis/server/storage/SnapshotManager.java

```diff
@@ -87,7 +89,7 @@ private FileChannel open(FileChunkProto chunk, File tmpSnapshotFile) throws IOEx
       }
       // create the temp snapshot file and put padding inside
       out = FileUtils.newFileChannel(tmpSnapshotFile, StandardOpenOption.WRITE, StandardOpenOption.CREATE);
-      digester = MD5Hash.newDigester();
+      digester.set(MD5Hash.getDigester());
```

Comment:
@133tosakarin , why changing back to `getDigester`?

Reply:
Sorry, I mistakenly thought that every time I use DIGESTER_FACTORY (), newDigest () will be called, so I want to try changing it to getDigester ().

I have changed back to newDigester.